packages/index.demo.tsx

```diff
@@ -0,0 +1,22 @@
+import './accordion/index.demo';
```

Comment:
You might not have rebased - this has been moved to **packages/blaze-elements**.